prefect_kubernetes/utilities.py

```diff
@@ -0,0 +1,74 @@
+""" Utilities for working with the Python Kubernetes API. """
+from pathlib import Path
+from typing import TypeVar, Union
+
+from kubernetes.client import models as k8s_models
+from prefect.infrastructure.kubernetes import KubernetesJob, KubernetesManifest
+
+base_types = {"str", "int", "float", "bool", "list[str]", "dict(str, str)", "object"}
```

Comment:
Curious if `dict(str, str)` is desired vs `dict[str, str]`?

Reply:
their convention uses parens ([e.g.](https://github.com/kubernetes-client/python/blob/9be097199a3cb5cb56bd5ce700fa3933189af407/kubernetes/client/models/v1_limit_range_item.py#L36))

Reply:
Okay thanks for confirming; maybe we can add a comment above to let future developers this isn't a typo?